location/templates/location.html

```diff
@@ -22,5 +23,41 @@
 
 {% block content %}
 <h1>{{object.address}}, {{object.city}}, {{object.state}}, {{object.zipcode}}</h1>
+    # After onclick "write review...",  should POST and display write review
+    # should automatically update the page
 {% leaflet_map "leaflet_map_main" %}
+    {% if object.review_set %}
+        <h2> Review: </h2>
+        <hr class="mt-0 mb-2 mt-2">
+        <div class="row h-65">
+            <div class="col">
+                <table class="table table-striped table-bordered table-hover table-sm results-table" width="100%">
+                    <thead>
+                        <tr>
+                            <th scope="col" class="th-sm">User</th>
```

Comment:
Should we display the user here or keep the user anonymous? Also, if we are displaying the user, full name (or first name) might be better than username.